images/windows/scripts/build/Configure-System.ps1

```diff
@@ -105,10 +105,10 @@ $servicesToDisable = @(
     'gupdate'
     'gupdatem'
     'StorSvc'
-)
-
-$servicesToDisable | Stop-SvcWithErrHandling
-$servicesToDisable | Set-SvcWithErrHandling -Arguments @{StartupType = "Disabled"}
+) | Get-Service -ErrorAction SilentlyContinue
+Stop-Service $servicesToDisable
+$servicesToDisable.WaitForStatus('Stopped', "00:01:00")
+$servicesToDisable | Set-Service -StartupType Disabled
```

Comment:
We had issue in the past, https://github.com/actions/runner-images/issues/8770, when similar approach wasn't handling non-existent services for custom builds. Is it ok to revert changes?

Reply:
There is a difference: I added `| Get-Service -ErrorAction SilentlyContinue` to handle that.